### PR TITLE
Add camel-grok wrapper

### PR DESCRIPTION
**Pull request by ozgurkisir-qlik**
Add camel-grok wrapper

### CODE REVIEW

**Comment by essobedo** [comment]
There is a conflict to solve

**Comment by ozgurkisir-qlik** [comment]
Conflict resolved.

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -839,6 +839,11 @@
         <bundle dependency='true'>mvn:org.threeten/threetenbp/1.6.8</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-google-pubsub/${project.version}</bundle>
     </feature>
+    <feature name='camel-grok' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <bundle dependency='true'>wrap:mvn:io.krakens/java-grok/${java-grogit k-version}</bundle>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]:
```suggestion
        <bundle dependency='true'>wrap:mvn:io.krakens/java-grok/${java-grok-version}</bundle>
```

File: features/src/main/feature/camel-features.xml [hunk 2]
```diff
@@ -839,6 +839,11 @@
         <bundle dependency='true'>mvn:org.threeten/threetenbp/1.6.8</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-google-pubsub/${project.version}</bundle>
     </feature>
+    <feature name='camel-grok' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <bundle dependency='true'>wrap:mvn:io.krakens/java-grok/${java-grok-version}</bundle>
+        <bundle>mvn:org.apache.camel.karaf/camel-grok/${project.version}</bundle>
+    </feature>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: Please put it back where it was, indeed it is ordered alphabetically

Comment (ozgurkisir-qlik) [reply features/src/main/feature/camel-features.xml]: Done

**Review by essobedo — APPROVED** [review]
LGTM

**Pull request merged**